packages/itinerary-body/src/otp-react-redux/route-description.js

```diff
@@ -43,5 +43,6 @@ export default function RouteDescription({ leg }) {
 }
 
 RouteDescription.propTypes = {
-  leg: legType.isRequired
+  leg: legType.isRequired,
+  LegIcon: PropTypes.elementType.isRequired
```

Comment:
Again, it seems like making this optional and using a default value is probably the right approach.

Reply:
If we do that, there should be a package-wide default value.